android/src/main/java/io/radar/react/RNRadarModule.java

```diff
@@ -146,8 +185,20 @@ public void requestPermissions(boolean background, final Promise promise) {
     }
 
     @ReactMethod
-    public void getLocation(final Promise promise) {
-        Radar.getLocation(new Radar.RadarLocationCallback() {
+    public void getLocation(String desiredAccuracy, final Promise promise) {
+        
+        RadarTrackingOptions.RadarTrackingOptionsDesiredAccuracy accuracyLevel = RadarTrackingOptions.RadarTrackingOptionsDesiredAccuracy.MEDIUM;        
```

Comment:
We can't default to medium confidence.  When `desiredAccuracy` is null we should call the version of the `Radar.getLocation()` function that doesn't take a `desiredAccuracy` param.

And it `desiredAccuracy` is a string but not a known value (high, medium, low) we should just promise.reject() with an error message

Reply:
@tjulien Got it. I will add test for those new functions and update 'getLocation' function.